skabase/SKALogger/SKALogger/SKALogger.py

```diff
@@ -128,33 +130,86 @@ def Log(self, argin):
         # PROTECTED REGION END #    //  SKALogger.Log
 
     @command(
-    dtype_in='str', 
+    dtype_in='DevVarLongStringArray', 
     doc_in="Central logging level for selected devices", 
     )
     @DebugIt()
     def SetCentralLoggingLevel(self, argin):
         # PROTECTED REGION ID(SKALogger.SetCentralLoggingLevel) ENABLED START #
-        pass
+        CentralLoggingLevel = argin[0][:]
+        CentralLoggingDevice = argin[1][:]
+        i = 0
+        while i < len(CentralLoggingLevel[:]):
+            self.info_stream("%s,%s", CentralLoggingLevel[i], CentralLoggingDevice[i])
+            dev1 = DeviceProxy(CentralLoggingDevice[i])
```

Comment:
replace the variable name from dev1 to something more appropriate.

Reply:
Replaced it with devProxy.